Makefile

```diff
@@ -1,18 +1,38 @@
-TARGETS := $(shell ls scripts)
+.PHONY: all build test validate package package-helm clean
 
-.dapper:
-	@echo Downloading dapper
-	@curl -sL https://releases.rancher.com/dapper/latest/dapper-$$(uname -s)-$$(uname -m) > .dapper.tmp
-	@@chmod +x .dapper.tmp
-	@./.dapper.tmp -v
-	@mv .dapper.tmp .dapper
+all: package
 
-$(TARGETS): .dapper
-	./.dapper $@
+build:
+	@echo "--- Building Webhook Image & Binary ---"
+	@bash -c 'source scripts/version && \
+	docker build \
+		--file package/Dockerfile \
+		--build-arg ARCH=$${ARCH} \
+		--build-arg VERSION=$${VERSION} \
+		--build-arg COMMIT=$${COMMIT} \
+		-t rancher/webhook:$${TAG} \
+		. && \
+	mkdir -p bin && \
+	CONTAINER_ID=$$(docker create rancher/webhook:$${TAG} echo) && \
+	docker cp $$CONTAINER_ID:/usr/bin/webhook ./bin/webhook && \
+	docker rm $$CONTAINER_ID'
```

Comment:
Alright so two things here.

1. We already have a way to build the binary only and have it copied into the host: the `binary` target in `package/Dockerfile`. The way to use it is to use docker buildx build with a few flags (look here how we do it in rancher: https://github.com/rancher/rancher/blob/ed11a0ddaa64d15f8dc2d219be625d37daa9ee06/dev-scripts/quick#L124-L131).

So no need to create the container, copy the binary (which might be for the wrong platform) back to the host.

2. The original `make build` command would only build the binary (see `scripts/build`). Now we're building both the binary and the image. I don't think this is necessary. We can stick to just building the image.